benchmarks/IO.py

```diff
@@ -33,12 +33,12 @@ def time_ak_write_read(N_per_locale, trials, dtype, path, seed):
     avgwrite = sum(writetimes) / trials
     avgread = sum(readtimes) / trials
 
-    print("Write times: min = {:.4f} sec, max = {:.4f} sec, avg = {:.4f} sec".format(min(writetimes), max(writetimes), avgwrite))
-    print("Read times : min = {:.4f} sec, max = {:.4f} sec, avg = {:.4f} sec".format(min(readtimes), max(readtimes), avgread))
+    print("write Average time = {:.4f} sec".format(avgwrite))
```

Comment:
I had included the min and max time because I/O rates are notoriously affected by buffering, but on second thought, the best solution for that is probably just to restrict it to a single trial.

Reply:
Yeah, makes sense -- lowering to 1 trial.